spec.html

```diff
@@ -132,7 +138,8 @@ <h1>AggregateError ( errors, message )</h1>
     1. Let _errorsIterator_ be ? GetMethod(_errors_, @@iterator).
     1. Let _errorsList_ ? IterableToList(_errors_, _errorsIterator_).
     1. Let _errorsArray_ be CreateArrayFromList(_errorsList_).
-    1. Perform ! CreateDataProperty(_O_, `"errors"`, _errorsArray_).
+    1. Let _errorsDesc_ be the PropertyDescriptor { [[Value]]: _errorsArray_, [[Writable]]: *true*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.
```

Comment:
maybe these two steps should be their own abstract op, so it can be reused in both places?

Reply:
I just found other places in current ecma262 spec text where such abstract operation could be reused (plus one usage for `message` property 3 lines below these two). It will need a lot arguments though. IMO duplication here is not a big problem but maybe worth a separate PR to the spec repo

Reply:
Fair point.

Reply:
"CreateBuiltinDataPropertyOrThrow"? Naming based on "CreateBuiltinFunction" and "CreateDataPropertyOrThrow", given that the majority of "built in" data properties are non-enumerable by default.